BurnOutSharp/FileType/PLJ.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Concurrent;
+using System.IO;
+using BurnOutSharp.Interfaces;
+using BurnOutSharp.Tools;
+
+namespace BurnOutSharp.FileType
+{
+    public class PLJ : IScannable
+    {
+        /// <inheritdoc/>
+        public bool ShouldScan(byte[] magic)
+        {
+            // https://www.iana.org/assignments/media-types/audio/vnd.everad.plj
+            if (magic.StartsWith(new byte?[] { 0xFF, 0x9D, 0x53, 0x4B }))
+                return true;
+
+            return false;
+        }
+
+        /// <inheritdoc/>
+        public ConcurrentDictionary<string, ConcurrentQueue<string>> Scan(Scanner scanner, string file)
+        {
+            if (!File.Exists(file))
+                return null;
+
+            using (var fs = File.OpenRead(file))
+            {
+                return Scan(scanner, fs, file);
+            }
+        }
+
+        /// <inheritdoc/>
+        public ConcurrentDictionary<string, ConcurrentQueue<string>> Scan(Scanner scanner, Stream stream, string file)
+        {
+            var protections = new ConcurrentDictionary<string, ConcurrentQueue<string>>();
+            try
+            {
+                Utilities.AppendToDictionary(protections, file, "PlayJ Audio File");
```

Comment:
Sort of, kinda... If you can put the signature check in here as well, I'd appreciate it. Even though any reasonable implementer of the library would use the `Scanner` class that invokes this separately, if you called this alone, it would always return that it's protected.